lib/matplotlib/widgets.py

```diff
@@ -1970,12 +1976,13 @@ def __init__(self, ax, onselect, direction, minspan=0, useblit=False,
         self.new_axes(ax)
 
         # Setup handles
-        props = dict(color=rectprops.get('facecolor', 'r'))
-        props.update(cbook.normalize_kwargs(handle_props, Line2D._alias_map))
+        _handle_props = dict(color=props.get('facecolor', 'r'))
```

Comment:
```suggestion
        handle_props_ = dict(color=props.get('facecolor', 'r'))
```
The standard notation for circumventing name clashes is a trailing underscore. Leading underscores are for "private" variables, which does not make sense for a local variable.

Reply:
Anyway, it's probably better to write this as 
```
        handle_props = {
            'color': props.get('facecolor', 'r'),
            **cbook.normalize_kwargs(handle_props, Line2D._alias_map)}
```